docs/orleans/quickstarts/build-your-first-orleans-app.md

```diff
@@ -49,7 +49,7 @@ At the end of the quickstart, you'll have an app that creates and handles redire
 
 1. Inside Visual Studio Code, open the [integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal).
 
-1. Change to the directory (`cd`) that will contain the project.
+1. Change directories (`cd`) to the project directory.
```

Comment:
"project directory" here could be confusing, since the following dotnet new command will create the project directory. I think the original "directory that will contain the project directory" would be clearer.